src/pages/workspace/categories/WorkspaceCategoriesPage.tsx

```diff
@@ -213,35 +226,35 @@ function WorkspaceCategoriesPage({route}: WorkspaceCategoriesPageProps) {
                     customText={translate('workspace.common.selected', {selectedNumber: selectedCategoriesArray.length})}
                     options={options}
                     isSplitButton={false}
-                    style={[isSmallScreenWidth && styles.flexGrow1, isSmallScreenWidth && styles.mb3]}
+                    style={[shouldUseNarrowLayout && styles.flexGrow1, shouldUseNarrowLayout && styles.mb3]}
                 />
             );
         }
 
         return (
-            <View style={[styles.w100, styles.flexRow, styles.gap2, isSmallScreenWidth && styles.mb3]}>
+            <View style={[styles.w100, styles.flexRow, styles.gap2, shouldUseNarrowLayout && styles.mb3]}>
                 {!PolicyUtils.hasAccountingConnections(policy) && (
                     <Button
                         medium
                         success
                         onPress={navigateToCreateCategoryPage}
                         icon={Expensicons.Plus}
                         text={translate('workspace.categories.addCategory')}
-                        style={[isSmallScreenWidth && styles.flex1]}
+                        style={[shouldUseNarrowLayout && styles.flex1]}
                     />
                 )}
                 <Button
                     medium
                     onPress={navigateToCategoriesSettings}
                     icon={Expensicons.Gear}
                     text={translate('common.settings')}
-                    style={[isSmallScreenWidth && styles.flex1]}
+                    style={[shouldUseNarrowLayout && styles.flex1]}
                 />
             </View>
         );
     };
 
-    const isLoading = !isOffline && policyCategories === undefined;
+    const isLoading = !isOffline && policyCategories === null;
```

Comment:
@ZhenjaHorbach why did you change it? Right now loading state is not working properly. See [here](https://github.com/Expensify/App/pull/44137#pullrequestreview-2153445711). Reverting this change fixes the issue